### ISSUE TITLE
Warnings in HA Core log on startup

### ISSUE THREAD

**Issue by donburch888** [nimroddolev/chime_tts#136]
## System Information

version | core-2024.5.4
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.12.2
os_name | Linux
os_version | 6.6.29-haos
arch | x86_64
timezone | Australia/Sydney
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 4882
Installed Version | 1.34.0
Stage | running
Available Repositories | 1462
Downloaded Repositories | 15

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | 3 June 2024 at 10:00 am
relayer_connected | true
relayer_region | ap-southeast-1
remote_enabled | true
remote_connected | true
alexa_enabled | false
google_enabled | false
remote_server | ap-southeast-1-1.ui.nabu.casa
certificate_status | ready
instance_id | 5ed6a2304786423a8e093715817678d7
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 12.3
-- | --
update_channel | stable
supervisor_version | supervisor-2024.05.1
agent_version | 1.6.0
docker_version | 25.0.5
disk_total | 62.3 GB
disk_used | 33.8 GB
healthy | true
supported | true
board | ova
supervisor_api | ok
version_api | ok
installed_addons | File editor (5.8.0), Samba share (12.3.1), Terminal & SSH (9.14.0), Mosquitto broker (6.4.0), Node-RED (17.0.12), Rhasspy Assistant (2.5.11), openWakeWord (1.10.0), Whisper (2.0.0), Piper (1.5.0), ESPHome (2024.5.0), Music Assistant BETA (2.1.0b0)

</details>

<details><summary>Dashboards</summary>

dashboards | 2
-- | --
resources | 8
views | 7
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 9 May 2024 at 10:54 pm
-- | --
current_recorder_run | 20 May 2024 at 9:47 am
estimated_db_size | 426.37 MiB
database_engine | sqlite
database_version | 3.44.2

</details>

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/nimroddolev/chime_tts/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

I have noticed two errors on startup in the HA Core log which seem to come from chime.tts.  I don't know if they are expected, or might be related to other issues I am experiencing (recently reported in separate issues by other users), so I am raising them separately. 

To aid debugging I have done a reinstall, using the v1.1.0.beta6 and tried to turn debugging on (in configuration.yaml - not sure if successful). 

Detected blocking call to import_module with args ('custom_components.chime_tts',) in /usr/src/homeassistant/homeassistant/loader.py, line 1041: ComponentProtocol, importlib.import_module(self.pkg_path) inside the event loop

/usr/local/lib/python3.12/asyncio/base_events.py:1989: RuntimeWarning: coroutine 'ChimeTTSQueueManager.queue_processor' was never awaited handle = None # Needed to break cycles when an exception occurs.

### Reproduction steps

1. installed ChimeTTS v1.1.0-beta6
2. rebooted HA
3. look in Home Assistant Core log

### Debug logs

```text
2024-05-20 10:07:49.665 WARNING (SyncWorker_6) [homeassistant.loader] We found a custom integration mass which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-05-20 10:07:49.665 WARNING (SyncWorker_6) [homeassistant.loader] We found a custom integration nodered which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-05-20 10:07:49.665 WARNING (SyncWorker_6) [homeassistant.loader] We found a custom integration chime_tts which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-05-20 10:07:49.666 WARNING (SyncWorker_6) [homeassistant.loader] We found a custom integration zha_toolkit which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-05-20 10:07:49.667 WARNING (SyncWorker_6) [homeassistant.loader] We found a custom integration tuya_local which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-05-20 10:07:49.667 WARNING (SyncWorker_6) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-05-20 10:07:49.668 WARNING (SyncWorker_6) [homeassistant.loader] We found a custom integration bureau_of_meteorology which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-05-20 10:07:50.773 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from rhasspy-0 (192.168.1.86). Requested URL: '/media/local/sounds/temp/chime_tts/cover.png'. (Music Player Daemon 0.21.5)
2024-05-20 10:07:50.813 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from rhasspy-0 (192.168.1.86). Requested URL: '/media/local/sounds/temp/chime_tts/cover.jpg'. (Music Player Daemon 0.21.5)
2024-05-20 10:07:50.841 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from rhasspy-0 (192.168.1.86). Requested URL: '/media/local/sounds/temp/chime_tts/cover.tiff'. (Music Player Daemon 0.21.5)
2024-05-20 10:07:50.942 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from rhasspy-0 (192.168.1.86). Requested URL: '/media/local/sounds/temp/chime_tts/cover.bmp'. (Music Player Daemon 0.21.5)
2024-05-20 10:07:51.689 DEBUG (ImportExecutor_0) [custom_components.chime_tts.queue_manager] Queue reset
2024-05-20 10:07:51.690 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to import_module with args ('custom_components.chime_tts',) in /usr/src/homeassistant/homeassistant/loader.py, line 1041: ComponentProtocol, importlib.import_module(self.pkg_path) inside the event loop
2024-05-20 10:07:51.691 DEBUG (MainThread) [custom_components.chime_tts.queue_manager] Queue reset
2024-05-20 10:07:51.692 INFO (MainThread) [custom_components.chime_tts] The Chime TTS integration is set up.
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts] Chime TTS Configuration Values:
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]  - queue_timeout: 60
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]  - tts_platform: ''
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]  - offset: 0
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]  - fade_transition: 500
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]  - temp_chimes_path: '/media/sounds/temp/chime_tts/chimes/'
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]  - temp_path: '/media/sounds/temp/chime_tts/'
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]  - www_path: '/config/www/chime_tts/'
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]  - media_dir: 'local'
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]  - custom_paths:
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]    - custom_chime_path_1: 
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]    - custom_chime_path_2: 
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]    - custom_chime_path_3: 
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]    - custom_chime_path_4: 
2024-05-20 10:07:51.692 DEBUG (MainThread) [custom_components.chime_tts]    - custom_chime_path_5: 
2024-05-20 10:07:51.696 WARNING (MainThread) [py.warnings] /usr/local/lib/python3.12/asyncio/base_events.py:1989: RuntimeWarning: coroutine 'ChimeTTSQueueManager.queue_processor' was never awaited
  handle = None  # Needed to break cycles when an exception occurs.

2024-05-20 10:07:52.466 WARNING (MainThread) [homeassistant.components.switch] Platform broadlink not ready yet: None; Retrying in background in 30 seconds
2024-05-20 10:07:52.658 WARNING (ImportExecutor_0) [py.warnings] /usr/local/lib/python3.12/site-packages/mashumaro/core/meta/types/unpack.py:182: UserWarning: music_assistant.common.models.config_entries.ConfigValueOption.value (typing.Union[str, int, float, bool, tuple[int, int], list[str], list[int], list[tuple[int, int]], None]): In the next release, data marked with Union type containing 'str' and 'bool' will be coerced to the value of the type specified first instead of passing it as is
  warnings.warn(

2024-05-20 10:07:52.665 WARNING (ImportExecutor_0) [py.warnings] /usr/local/lib/python3.12/site-packages/mashumaro/core/meta/types/unpack.py:182: UserWarning: music_assistant.common.models.config_entries.ConfigEntry.default_value (typing.Union[str, int, float, bool, tuple[int, int], list[str], list[int], list[tuple[int, int]], None]): In the next release, data marked with Union type containing 'str' and 'bool' will be coerced to the value of the type specified first instead of passing it as is
  warnings.warn(

2024-05-20 10:07:52.670 WARNING (ImportExecutor_0) [py.warnings] /usr/local/lib/python3.12/site-packages/mashumaro/core/meta/types/unpack.py:182: UserWarning: music_assistant.common.models.config_entries.ConfigEntry.value (typing.Union[str, int, float, bool, tuple[int, int], list[str], list[int], list[tuple[int, int]], None]): In the next release, data marked with Union type containing 'str' and 'bool' will be coerced to the value of the type specified first instead of passing it as is
  warnings.warn(

2024-05-20 10:07:52.802 WARNING (MainThread) [homeassistant.helpers.frame] Detected that custom integration 'hacs' accesses hass.components.frontend. This is deprecated and will stop working in Home Assistant 2024.9, it should be updated to import functions used from frontend directly at custom_components/hacs/frontend.py, line 68: hass.components.frontend.async_register_built_in_panel(, please create a bug report at https://github.com/hacs/integration/issues
2024-05-20 10:08:49.801 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: {('kodi', '75d3cbc3e2f4cf6023378ecd2522995c'): 2998.11789393}
2024-05-20 10:09:49.859 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: {('kodi', '75d3cbc3e2f4cf6023378ecd2522995c'): 2998.11789393}
2024-05-20 10:10:05.016 WARNING (MainThread) [homeassistant.helpers.frame] Detected that custom integration 'hacs' accesses hass.helpers.event. This is deprecated and will stop working in Home Assistant 2024.11, it should be updated to import functions used from event directly at custom_components/hacs/base.py, line 630: self.hass.helpers.event.async_track_time_interval(, please create a bug report at https://github.com/hacs/integration/issues
2024-05-20 10:12:24.267 WARNING (MainThread) [homeassistant.components.kodi.media_player] Unable to connect to Kodi via websocket
```

### Diagnostics dump

where do I find the diagnostics dump ?

**Comment by def1149** [comment]
I have similar warnings  on startup. 
HAOS 2024.5.5
Chime TTS v1.1.0-beta6
```
2024-05-25 12:29:59.675 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to import_module with args ('custom_components.chime_tts',) in /usr/src/homeassistant/homeassistant/loader.py, line 1046: ComponentProtocol, importlib.import_module(self.pkg_path) inside the event loop
2024-05-25 12:29:59.681 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=chime_tts, service=say>
2024-05-25 12:29:59.682 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=chime_tts, service=say_url>
2024-05-25 12:29:59.682 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=chime_tts, service=clear_cache>
2024-05-25 12:29:59.683 WARNING (MainThread) [py.warnings] /usr/local/lib/python3.12/asyncio/base_events.py:1989: RuntimeWarning: coroutine 'ChimeTTSQueueManager.queue_processor' was never awaited
  handle = None  # Needed to break cycles when an exception occurs.

2024-05-25 12:29:59.701 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event component_loaded[L]: component=chime_tts>
```

**Comment by nimroddolev** [comment]
I have released a new version, v1.1.0-beta10, which should address the issue.

**Comment by pepe59** [comment]
I don't know if this is related, but after updating HA version 2024.6 and updating Chime 1.1.0-beta10, I get errors every time I activate voice notification

```
Logger: homeassistant.util.loop
Zdroj: util/loop.py:84
První výskyt: 20:00:01 (18 výskyty)
Naposledy logováno: 21:00:05

Detected blocking call to open inside the event loop by custom integration 'chime_tts' at custom_components/chime_tts/helpers/filesystem.py, line 157: audio.export(audio_full_path, format="mp3") (offender: /usr/local/lib/python3.12/site-packages/pydub/audio_segment.py, line 962: with open(os.devnull, 'rb') as devnull:), please create a bug report at https://github.com/nimroddolev/chime_tts/issues Traceback (most recent call last): File "<frozen runpy>", line 198, in _run_module_as_main File "<frozen runpy>", line 88, in _run_code File "/usr/src/homeassistant/homeassistant/__main__.py", line 223, in <module> sys.exit(main()) File "/usr/src/homeassistant/homeassistant/__main__.py", line 209, in main exit_code = runner.run(runtime_conf) File "/usr/src/homeassistant/homeassistant/runner.py", line 190, in run return loop.run_until_complete(setup_and_run_hass(runtime_config)) File "/usr/local/lib/python3.12/asyncio/base_events.py", line 672, in run_until_complete self.run_forever() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 639, in run_forever self._run_once() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 1988, in _run_once handle._run() File "/usr/local/lib/python3.12/asyncio/events.py", line 88, in _run self._context.run(self._callback, *self._args) File "/config/custom_components/chime_tts/queue_manager.py", line 38, in async_process_queue result = await asyncio.wait_for( File "/usr/local/lib/python3.12/asyncio/tasks.py", line 520, in wait_for return await fut File "/config/custom_components/chime_tts/__init__.py", line 161, in async_say_execute return await async_prepare_media(hass, params, options, media_players_array, is_say_url, start_time) File "/config/custom_components/chime_tts/__init__.py", line 255, in async_prepare_media audio_dict = await async_get_playback_audio_path(params, options) File "/config/custom_components/chime_tts/__init__.py", line 695, in async_get_playback_audio_path output_audio = await async_process_segments(hass, File "/config/custom_components/chime_tts/__init__.py", line 916, in async_process_segments tts_audio = helpers.change_speed_of_audiosegment(tts_audio, segment_tts_speed, temp_folder) File "/config/custom_components/chime_tts/helpers/helpers.py", line 506, in change_speed_of_audiosegment return self.ffmpeg_convert_from_audio_segment(audio_segment=audio_segment, File "/config/custom_components/chime_tts/helpers/helpers.py", line 358, in ffmpeg_convert_from_audio_segment temp_audio_file = filesystem_helper.save_audio_to_folder(audio=audio_segment, File "/config/custom_components/chime_tts/helpers/filesystem.py", line 157, in save_audio_to_folder audio.export(audio_full_path, format="mp3")
Detected blocking call to open inside the event loop by custom integration 'chime_tts' at custom_components/chime_tts/helpers/helpers.py, line 374: ret_val = AudioSegment.from_file(str(converted_audio_file)) (offender: /usr/local/lib/python3.12/site-packages/pydub/utils.py, line 60: fd = open(fd, mode=mode)), please create a bug report at https://github.com/nimroddolev/chime_tts/issues Traceback (most recent call last): File "<frozen runpy>", line 198, in _run_module_as_main File "<frozen runpy>", line 88, in _run_code File "/usr/src/homeassistant/homeassistant/__main__.py", line 223, in <module> sys.exit(main()) File "/usr/src/homeassistant/homeassistant/__main__.py", line 209, in main exit_code = runner.run(runtime_conf) File "/usr/src/homeassistant/homeassistant/runner.py", line 190, in run return loop.run_until_complete(setup_and_run_hass(runtime_config)) File "/usr/local/lib/python3.12/asyncio/base_events.py", line 672, in run_until_complete self.run_forever() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 639, in run_forever self._run_once() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 1988, in _run_once handle._run() File "/usr/local/lib/python3.12/asyncio/events.py", line 88, in _run self._context.run(self._callback, *self._args) File "/config/custom_components/chime_tts/queue_manager.py", line 38, in async_process_queue result = await asyncio.wait_for( File "/usr/local/lib/python3.12/asyncio/tasks.py", line 520, in wait_for return await fut File "/config/custom_components/chime_tts/__init__.py", line 161, in async_say_execute return await async_prepare_media(hass, params, options, media_players_array, is_say_url, start_time) File "/config/custom_components/chime_tts/__init__.py", line 255, in async_prepare_media audio_dict = await async_get_playback_audio_path(params, options) File "/config/custom_components/chime_tts/__init__.py", line 695, in async_get_playback_audio_path output_audio = await async_process_segments(hass, File "/config/custom_components/chime_tts/__init__.py", line 916, in async_process_segments tts_audio = helpers.change_speed_of_audiosegment(tts_audio, segment_tts_speed, temp_folder) File "/config/custom_components/chime_tts/helpers/helpers.py", line 506, in change_speed_of_audiosegment return self.ffmpeg_convert_from_audio_segment(audio_segment=audio_segment, File "/config/custom_components/chime_tts/helpers/helpers.py", line 374, in ffmpeg_convert_from_audio_segment ret_val = AudioSegment.from_file(str(converted_audio_file))
Detected blocking call to open inside the event loop by custom integration 'chime_tts' at custom_components/chime_tts/helpers/filesystem.py, line 144: audio.export(audio_full_path, format="mp3") (offender: /usr/local/lib/python3.12/site-packages/pydub/utils.py, line 60: fd = open(fd, mode=mode)), please create a bug report at https://github.com/nimroddolev/chime_tts/issues Traceback (most recent call last): File "<frozen runpy>", line 198, in _run_module_as_main File "<frozen runpy>", line 88, in _run_code File "/usr/src/homeassistant/homeassistant/__main__.py", line 223, in <module> sys.exit(main()) File "/usr/src/homeassistant/homeassistant/__main__.py", line 209, in main exit_code = runner.run(runtime_conf) File "/usr/src/homeassistant/homeassistant/runner.py", line 190, in run return loop.run_until_complete(setup_and_run_hass(runtime_config)) File "/usr/local/lib/python3.12/asyncio/base_events.py", line 672, in run_until_complete self.run_forever() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 639, in run_forever self._run_once() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 1988, in _run_once handle._run() File "/usr/local/lib/python3.12/asyncio/events.py", line 88, in _run self._context.run(self._callback, *self._args) File "/config/custom_components/chime_tts/queue_manager.py", line 38, in async_process_queue result = await asyncio.wait_for( File "/usr/local/lib/python3.12/asyncio/tasks.py", line 520, in wait_for return await fut File "/config/custom_components/chime_tts/__init__.py", line 161, in async_say_execute return await async_prepare_media(hass, params, options, media_players_array, is_say_url, start_time) File "/config/custom_components/chime_tts/__init__.py", line 255, in async_prepare_media audio_dict = await async_get_playback_audio_path(params, options) File "/config/custom_components/chime_tts/__init__.py", line 717, in async_get_playback_audio_path new_audio_file = filesystem_helper.save_audio_to_folder(output_audio, _data.get(initial_save_folder_key, None)) File "/config/custom_components/chime_tts/helpers/filesystem.py", line 144, in save_audio_to_folder audio.export(audio_full_path, format="mp3")
Detected blocking call to open inside the event loop by custom integration 'chime_tts' at custom_components/chime_tts/helpers/filesystem.py, line 144: audio.export(audio_full_path, format="mp3") (offender: /usr/local/lib/python3.12/site-packages/pydub/audio_segment.py, line 962: with open(os.devnull, 'rb') as devnull:), please create a bug report at https://github.com/nimroddolev/chime_tts/issues Traceback (most recent call last): File "<frozen runpy>", line 198, in _run_module_as_main File "<frozen runpy>", line 88, in _run_code File "/usr/src/homeassistant/homeassistant/__main__.py", line 223, in <module> sys.exit(main()) File "/usr/src/homeassistant/homeassistant/__main__.py", line 209, in main exit_code = runner.run(runtime_conf) File "/usr/src/homeassistant/homeassistant/runner.py", line 190, in run return loop.run_until_complete(setup_and_run_hass(runtime_config)) File "/usr/local/lib/python3.12/asyncio/base_events.py", line 672, in run_until_complete self.run_forever() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 639, in run_forever self._run_once() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 1988, in _run_once handle._run() File "/usr/local/lib/python3.12/asyncio/events.py", line 88, in _run self._context.run(self._callback, *self._args) File "/config/custom_components/chime_tts/queue_manager.py", line 38, in async_process_queue result = await asyncio.wait_for( File "/usr/local/lib/python3.12/asyncio/tasks.py", line 520, in wait_for return await fut File "/config/custom_components/chime_tts/__init__.py", line 161, in async_say_execute return await async_prepare_media(hass, params, options, media_players_array, is_say_url, start_time) File "/config/custom_components/chime_tts/__init__.py", line 255, in async_prepare_media audio_dict = await async_get_playback_audio_path(params, options) File "/config/custom_components/chime_tts/__init__.py", line 717, in async_get_playback_audio_path new_audio_file = filesystem_helper.save_audio_to_folder(output_audio, _data.get(initial_save_folder_key, None)) File "/config/custom_components/chime_tts/helpers/filesystem.py", line 144, in save_audio_to_folder audio.export(audio_full_path, format="mp3")
Detected blocking call to open inside the event loop by custom integration 'chime_tts' at custom_components/chime_tts/__init__.py, line 733: duration = len(AudioSegment.from_file(new_audio_file)) / 1000.0 (offender: /usr/local/lib/python3.12/site-packages/pydub/utils.py, line 60: fd = open(fd, mode=mode)), please create a bug report at https://github.com/nimroddolev/chime_tts/issues Traceback (most recent call last): File "<frozen runpy>", line 198, in _run_module_as_main File "<frozen runpy>", line 88, in _run_code File "/usr/src/homeassistant/homeassistant/__main__.py", line 223, in <module> sys.exit(main()) File "/usr/src/homeassistant/homeassistant/__main__.py", line 209, in main exit_code = runner.run(runtime_conf) File "/usr/src/homeassistant/homeassistant/runner.py", line 190, in run return loop.run_until_complete(setup_and_run_hass(runtime_config)) File "/usr/local/lib/python3.12/asyncio/base_events.py", line 672, in run_until_complete self.run_forever() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 639, in run_forever self._run_once() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 1988, in _run_once handle._run() File "/usr/local/lib/python3.12/asyncio/events.py", line 88, in _run self._context.run(self._callback, *self._args) File "/config/custom_components/chime_tts/queue_manager.py", line 38, in async_process_queue result = await asyncio.wait_for( File "/usr/local/lib/python3.12/asyncio/tasks.py", line 520, in wait_for return await fut File "/config/custom_components/chime_tts/__init__.py", line 161, in async_say_execute return await async_prepare_media(hass, params, options, media_players_array, is_say_url, start_time) File "/config/custom_components/chime_tts/__init__.py", line 255, in async_prepare_media audio_dict = await async_get_playback_audio_path(params, options) File "/config/custom_components/chime_tts/__init__.py", line 733, in async_get_playback_audio_path duration = len(AudioSegment.from_file(new_audio_file)) / 1000.0
```

**Comment by nimroddolev** [comment]
Thank you @pepe59. I have found and fixed these issues and they'll be part of the next release.
Do you experience any issues as a result of these warnings?

**Comment by pepe59** [comment]
No there are no problems just these warnings in the HA log.
Thank you for a great component works perfectly.

**Comment by nimroddolev** [comment]
Thank you 🙏 I’m glad to hear it’s working well for you!

I just released another beta version, v1.1.0-beta11, which includes fixes for the cause of the warnings you shared.

**Comment by pepe59** [comment]
I can confirm that the warning issue is fixed v1.1.0-beta11.
Thanks

**Comment by fragolinux** [comment]
same problems for me... how to change the HACS repo to use latest beta? I'm on 1.0.0

**Comment by fragolinux** [comment]
confirm working on 1.1.0-beta13

**Comment by nimroddolev** [comment]
The fix has been included in the latest stable release, v1.1.0.
